examples/packages/parcel/.babelrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file will never be read because this is monorepo ([Babel's rule](https://babeljs.io/docs/en/config-files#monorepos), not Parcel-specific)